cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/instance/AvailabilitySetNameService.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.cloud.model.instance;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class AvailabilitySetNameService {
```

Comment:
Is this going to be used for other cloud providers as well? Just wondering why you didn't put it with the other Azure stuff.

Reply:
This logic used to be hardcoded in `LoadBalancerConfigService`. I needed to be able to generate the AS name in `AzureLoadBalancerMetadataCollector` as well, but due to module dependencies I couldn't import `LoadBalancerConfigService` into that class. So I moved it here so that both of those classes could import it and use the logic.

Reply:
Ah okay that makes sense, thanks for letting me know.